NEWS.md

```diff
@@ -44,27 +47,27 @@
 
 # ellmer 0.2.1
 
-* When you save a `Chat` object to disk, API keys are automatically redacted. 
```

Comment:
Can you do this once in a different PR? I think I agree with this IDE setting in principle, but I turn it off because I find it creates too many spurious diffs when I'm dropping into random projects, and thus my own projects end up with trailing whitespace.

Reply:
Shoot, sorry! I usually notice and go back and revert the extra changes, but I missed this one. I'll take this out of this PR and clean up NEWS separately.

Reply:
Ok gotcha; ignore my comment in the other PR then.